.goreleaser.yml

```diff
@@ -28,7 +28,7 @@ archives:
 checksum:
   name_template: 'checksums.txt'
 snapshot:
-  name_template: "{{ incpatch .Version }}-next"
+  name_template: "{{ incpatch .Version }}-ng"
```

Comment:
What is this change? will the binary be called furyctl-ng?

Reply:
TBD - this particular change is just to align the nomenclature. We'll address this when we'll think about the release process for ng vs legacy